### PR TITLE
add open-vxlan-port patch

### PR DESCRIPTION
**Pull request by benjamreis**

### CODE REVIEW

File: SPECS/xapi.spec [hunk 1]
```diff
@@ -551,7 +555,7 @@ Coverage files from unit tests
 - CP-29757: Add new VDI_IS_ENCRYPTED exception
 
 * Tue Nov 27 2018 Christian Lindig <christian.lindig@citrix.com> - 1.137.0-1
-- CP-30039: Generate automatically the release and class files for 
+- CP-30039: Generate automatically the release and class files for
```
Comment (stormi) [inline SPECS/xapi.spec]: Tune your editor to not remove trailing spaces on text that we're not the upstream for :)

File: SPECS/xapi.spec [hunk 2]
```diff
@@ -436,6 +437,9 @@ Coverage files from unit tests
 %endif
 
 %changelog
+* Tue Jun 25 2019 Benjamin Reis <benjamin.reis@vates.fr> - 1.160.1-1.2
```
Comment (stormi) [inline SPECS/xapi.spec]: Changing the release number here is not enough, you need to change it in the Release: tag above too.

File: SOURCES/xapi-1.160.1-open-vxlan-port-for-sdn-controller.XCP-ng.patch [hunk 3]
```diff
@@ -0,0 +1,23 @@
+diff --git c/scripts/plugins/openvswitch-config-update w/scripts/plugins/openvswitch-config-update
```
Comment (stormi) [inline SOURCES/xapi-1.160.1-open-vxlan-port-for-sdn-controller.XCP-ng.patch]: Add a comment on top of patch explaining what it does and why it's needed.

**Comment by benjamreis** [comment]
https://koji.xcp-ng.org//taskinfo?taskID=8939

**Comment by stormi** [comment]
Looks good. Please squash the commits and then I'll merge and submit the build.

**Comment by benjamreis** [comment]
https://koji.xcp-ng.org//taskinfo?taskID=8942

Testing created RPM

**Comment by benjamreis** [comment]
Tests successful :+1:

**Comment by stormi** [comment]
Build sent to `v8.0-base` git+https://github.com/xcp-ng-rpms/xapi.git?#ee05c6f8c0dd24e22e1be588cc37a5de796c8f21

**Pull request merged**